client/src/main/java/io/apicurio/registry/rest/client/RegistryClient.java

```diff
@@ -44,7 +45,7 @@
  * <p>
  * Rest client compatible with the Registry API V2
  */
-public interface RegistryClient {
+public interface RegistryClient extends Closeable {
```

Comment:
Not `AutoCloseable`?

Reply:
+1

Reply:
aaah it's because `AutoCloseable` close method does not throw IOException

Reply:
Yes, that was my only reason.